### PR TITLE
[vcpkg edit] Simplify execution of VS Code

### PR DESCRIPTION
**Pull request by strega-nil**
**Describe the pull request**

- What does your PR fix? Fixes #

- Which triplets are supported/not supported? Have you updated the CI baseline?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

**Review by ras0219 — APPROVED** [review]

File: toolsrc/src/vcpkg/commands.edit.cpp [hunk 1]
```diff
@@ -263,8 +263,7 @@ namespace vcpkg::Commands::Edit
         if (editor_exe == "Code.exe" || editor_exe == "Code - Insiders.exe")
         {
             // note that we are invoking cmd silently but Code.exe is relaunched from there
```
Comment (ras0219) [inline toolsrc/src/vcpkg/commands.edit.cpp:265]: This comment seems outdated :)

**Comment by BillyONeal** [comment]
This relaunching with cmd was to protect VS Code from `cmd_execute_background`s `CREATE_NO_WINDOW` I thought?

**Comment by strega-nil** [comment]
@BillyONeal Not really. If you want more information you can ask on Teams.

**Comment by BillyONeal** [comment]
>> This relaunching with cmd was to protect VS Code from `cmd_execute_background`s `CREATE_NO_WINDOW` I thought?
> @BillyONeal Not really. If you want more information you can ask on Teams.

I just tested this and I'm seeing no window. And a ton of spew in the console.

image

**Review by BillyONeal — CHANGES_REQUESTED** [review]
This breaks launching code in my testing.

**Comment by BillyONeal** [comment]
Note that to demonstrate the problem you need to not have an already running copy of VS Code somewhere.

**Comment by strega-nil** [comment]
@BillyONeal ah, that's why we were having issues.

**Comment by ras0219-msft** [comment]
@strega-nil Ping on this

**Comment by strega-nil** [comment]
Closing since clearly this breaks something.

**Pull request closed without merging**